src/Pages/OnboardingRoleSelection.tsx

```diff
@@ -0,0 +1,65 @@
+import { RoleSelectOnboardOne } from '../components/RoleSelectOnboardOne/RoleSelectOnboardOne'
+
+const footerLinks = [
+    {
+      summary: 'Madamada',
+      links: [
+        { linkContent: 'about', linkUrl: '#about' },
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '1',
+    },
+    {
+      summary: 'Madamada',
+      links: [
+        { linkContent: 'about', linkUrl: '#about' },
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '2',
+    },
+    {
+      summary: 'Madamada',
+      links: [
+        { linkContent: 'about', linkUrl: '#about' },
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '3',
+    },
+    {
+      summary: 'Madamada',
+      links: [
+        { linkContent: 'about', linkUrl: '#about' },
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '4',
+    },
+    {
+      summary: 'Madamada',
+      links: [
+        { linkContent: 'about', linkUrl: '#about' },
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '5',
+    },
+]
+
+
+function OnboardingRoleSelection() {
```

Comment:
Lets rename this to OnboardingPage.tsx as it will also serve for register form later